### PR TITLE
support relations, array result of queryAll and long numbers

### PR DESCRIPTION
**Pull request by geremiasseq**
relations like "firstRel.secondRel.n.propertyOfLastRel"
array = Yii::app()->db->createCommand("SELECT \* FROM t")->queryAll();
long numbers for example bank occount number or values that start with "0"

### CODE REVIEW

File: XlsExporter.php [hunk 1]
```diff
@@ -45,53 +45,83 @@ private static function sendHeader($filename, $length, $type = 'octet-stream')
     /**
      * Private method to create xls string from active record data set
      *
-     * @param $data - Active record data set
+     * @param $data - CActiveRecord or Array
+     * Ejemplos: 1.CActiveRecord: $data = Model::model()->findAll(new CDbCriteria()->condition = "column > $param");
```
Comment (faustow) [inline XlsExporter.php]: "Ejemplos" should be translated to English to keep the language consistent.

File: XlsExporter.php [hunk 2]
```diff
@@ -45,53 +45,83 @@ private static function sendHeader($filename, $length, $type = 'octet-stream')
...
+            . '<meta http-equiv="content-type" content="text/html; charset=utf-8">' . self::CRLF
+            . '<style> .text{ mso-number-format:\@; } </style>' . self::CRLF
+            . '</head>' . self::CRLF
+            . '<body style="text-align:center">' . self::CRLF;
 
         if ($title)
             $string .= "<b>$title</b><br /><br />" . self::CRLF
-            . Yii::t('main', 'Exported '.$type) . ': ' . count($data) . '<br />' . self::CRLF
-            . Yii::t('main', 'Export date') . ': ' . Yii::app()->dateFormatter->formatDateTime($_SERVER['REQUEST_TIME']) . '<br /><br />' . self::CRLF;
+                . Yii::t('main', 'Total de registros '.$type) . ': ' . count($data) . '<br />' . self::CRLF
```
Comment (faustow) [inline XlsExporter.php]: Please translate the spanish in here too.

File: XlsExporter.php [hunk 3]
```diff
@@ -45,53 +45,83 @@ private static function sendHeader($filename, $length, $type = 'octet-stream')
...
+                . Yii::t('main', 'Fecha') . ': ' . Yii::app()->dateFormatter->formatDateTime($_SERVER['REQUEST_TIME']) . '<br /><br />' . self::CRLF;
```
Comment (faustow) [inline XlsExporter.php]: Please translate the spanish in here too.

File: XlsExporter.php (hunk 1 above)
Comment (geremiasseq) [inline XlsExporter.php]: Done!

**Comment by geremiasseq** [comment]
the text has been fully translated

**Pull request merged**